src/apps/dictionaries/pages/CreateDictionaryPage.tsx

```diff
@@ -26,6 +27,7 @@ interface Props {
   ) => void;
   loading: boolean;
   newDictionary?: APIDictionary;
+  resetCreateDictionaryAction: () => void;
```

Comment:
```suggestion
  resetCreateDictionary: () => void;
```

Reply:
The name in the `Props` interface needs to match the name on line 39 and the name in `mapActionsToProps`.